app/api/data.py

```diff
@@ -354,13 +354,22 @@ def edit_core_data_from_states_daily():
             notify_slack_error(error, 'edit_core_data_from_states_daily')
             return flask.jsonify(error), 400
 
+        valid, unknown = CoreData.valid_fields_checker(core_data_dict)
+        if not valid:
+            # there are no fields to add/update
+            flask.current_app.logger.debug('Got row without updates, skipping. %r' % core_data_dict)
+            continue
+
+        if unknown:
+            # report unknown fields, we won't fail the request, but should at least log
+            flask.current_app.logger.warning('Got row with unkwon field updates. %r' % core_data_dict)
```

Comment:
nit: unknown :) also, I don't have strong feelings about this, but the `debug` log above might be worth getting as an `info` log?